frontend/src/bundles/studio/components/player-controls/components/control/control.tsx

```diff
@@ -0,0 +1,36 @@
+import { type ElementType } from 'react';
+
+import {
+    Icon,
+    IconButton,
+    Tooltip,
+} from '~/bundles/common/components/components.js';
+
+type Properties = {
+    label: string;
+    size: string;
```

Comment:
The size prop could be change to a custom type as it only accepts "lg", "md", "sm" or "xs" per chakra documentation

You could define a type
type Size = 'lg' | 'md' | 'sm' | 'xs'

and in Properties:
size: Size;